WreckEngine/External.h

```diff
@@ -6,41 +6,50 @@
 #include "Random.h"
 
 struct Window {
-	static GLFWwindow* window;
-	static int width, height;
-	static float aspect;
+    static GLFWwindow* window;
+    static int width, height;
+    static int frameWidth, frameHeight;
+    static float aspect;
+    static int cursorMode;
 
-	static inline void update() { glfwGetWindowSize(window, &width, &height); aspect = (float)width / (float)height; }
-	static inline int getKey(const int keyCode) { return glfwGetKey(window, keyCode); }
+    // These two are purely for convenience with ImGui
+    static vec2 size;
+    static vec2 frameScale;
```

Comment:
Do you mean "are `size` and `frameScale` the same general thing"? Because if so, no they're not.